_config.yml

```diff
@@ -1,56 +1,56 @@
-# Welcome to Jekyll!
-#
-# This config file is meant for settings that affect your whole blog, values
-# which you are expected to set up once and rarely need to edit after that.
-# For technical reasons, this file is *NOT* reloaded automatically when you use
-# 'jekyll serve'. If you change this file, please restart the server process.
-#
-# ,--------.      ,--.   ,--. ,--.
-# '--.  .--',---.  \  `.'  /,-'  '-.
-#    |  |  | .-. :  .'    \ '-.  .-'
-#    |  |  \   --. /  .'.  \  |  |
-#    `--'   `----''--'   '--' `--'
-
-
-
-
 ## => Site Settings
 ##############################
-text_skin: default # "default" (default), "dark", "forest", "ocean", "chocolate", "orange"
-highlight_theme: default # "default" (default), "tomorrow", "tomorrow-night", "tomorrow-night-eighties", "tomorrow-night-blue", "tomorrow-night-bright"
-url     : # the base hostname & protocol for your site e.g. https://www.someone.com
-baseurl : # does not include hostname
+text_skin: default 
+highlight_theme: default 
+url     : /
+baseurl : /
 title   : learn-pip-trends
-description: > # this means to ignore newlines until "Language & timezone"
+tagline : Learn everything about Python
+description: > 
   Learn everything about Python
+author : Ashok Tankala
+twitter:
+  username: piptrends
+  card: summary
+facebook:
```

Comment:
What is the advantage of facebook here. Check once.

Reply:
This is for share preview(Open Graph) on Facebook.